core/app/models/spree/calculator/flat_fee.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require_dependency 'spree/calculator'
```

Comment:
`require_dependency` has been [deprecated](https://github.com/rails/rails/blob/main/activesupport/lib/active_support/dependencies/require_dependency.rb). Anyway, why do we need it? 🤔